op-acceptor/ui/tree.go

```diff
@@ -0,0 +1,126 @@
+package ui
+
+import (
+	"strings"
+	"unicode/utf8"
+)
+
+// Tree hierarchy symbols using box drawing characters
+const (
+	// Basic tree connectors
+	TreeBranch     = "├── " // Branch connector (tee right + horizontal line + space)
+	TreeLastBranch = "└── " // Last branch connector (bottom left corner + horizontal line + space)
+	TreeVertical   = "│"    // Vertical line for continuing hierarchy
+
+	// Spacing patterns for different indentation levels
+	TreeContinue   = "│   "     // Vertical line + 3 spaces (parent has more siblings)
+	TreeIndent     = "    "     // 4 spaces (parent was last, no vertical line needed)
+	TreeDeepIndent = "│       " // Vertical line + 7 spaces (deeper nesting)
+	TreeMultiLevel = "│   │   " // Multiple vertical lines for complex hierarchy
+
+	// Box drawing characters for borders/containers
+	BoxTopLeft     = "┌"
+	BoxTopRight    = "┐"
+	BoxBottomLeft  = "└"
+	BoxBottomRight = "┘"
+	BoxVertical    = "│"
+	BoxHorizontal  = "─"
+	BoxTeeRight    = "├"
+	BoxTeeLeft     = "┤"
+
+	// Composite patterns for common use cases
+	TreeSubTestBranch     = "│       ├──"         // For subtests under tests
+	TreeSubTestLastBranch = "│       └──"         // For last subtest under tests
+	TreeSubTestError      = "│       │       └──" // For errors under subtests
+
+	// Suite-specific patterns (with additional indentation)
+	SuiteBranch        = "    ├── "                // Branch under suite
+	SuiteLastBranch    = "    └── "                // Last branch under suite
+	SuiteContinue      = "    │   "                // Continue line under suite
+	SuiteSubTestBranch = "    │       ├──"         // Subtest under suite test
```

Comment:
I haven't looked at the details, but the existence of this constant seems to suggest we're not going to be able to represent deeper levels of subtests still?

Reply:
When I re-checked this, I found that, indeed, the data layer supported it but not the presentation layer. Updated it and added a test as well.